data_diff/databases/bigquery.py

```diff
@@ -62,7 +65,7 @@ def import_bigquery_service_account_impersonation():
 @attrs.define(frozen=False)
 class Mixin_MD5(AbstractMixin_MD5):
     def md5_as_int(self, s: str) -> str:
-        return f"cast(cast( ('0x' || substr(TO_HEX(md5({s})), 18)) as int64) as numeric)"
```

Comment:
The number of digits used from the checksum was hardcoded and not the same as in other databases 🫠